### PR TITLE
WD-16838 Rebranded /landscape/managed page

### PR DESCRIPTION
**Pull request by muhammad-ali-pk**
## Done

- Rebranded page

## QA

- View the site  in your web browser at: https://ubuntu-com-14580.demos.haus/landscape/managed
    - Be sure to test on mobile, tablet and desktop screen sizes
- Check that it matches the following:
   - [Design](https://www.figma.com/design/btsxPLFKGPWDlbQ4rjR4jZ/25.04-Landscape-Rebranding?node-id=4015-2226&t=2riHUPsatn9YCZDu-0)
   - [CopyDoc](https://docs.google.com/document/d/1MycPWcmMveCZxrGtn7ASc0myWNxPzLTciHfQIA6kPZ4/edit?tab=t.0#)

## Issue / Card

Fixes #[WD-16838](https://warthogs.atlassian.net/browse/WD-16838)

[WD-16838]: https://warthogs.atlassian.net/browse/WD-16838?atlOrigin=eyJpIjoiNWRkNTljNzYxNjVmNDY3MDlhMDU5Y2ZhYzA5YTRkZjUiLCJwIjoiZ2l0aHViLWNvbS1KU1cifQ

### CODE REVIEW

**Comment by mattea-turic** [comment]
Thanks @muhammad-ali-pk ! Only a couple of things from me:

- [ ] For the section "Bring your own identity and access management platform", the image should have the grey background – I forgot to apply that back on figma after exporting the image, so mb!
- [ ] I'd also say to hide that image on smaller screens as it's there's too much detail for it to work unless you're viewing on a desktop (or large tablet)
- [ ] You forgot this section: "Mirror Ubuntu repositories, and distribute software internally"

**Comment by eliman11** [comment]
Thanks @muhammad-ali-pk, this looks great! Just a few quick comments:

- [ ] Link asterisk(*) in the pricing block to the live Managed Apps page instead of the copydoc for that page, also added as a suggestion in the  copydoc - https://ubuntu.com/managed/apps 
- [ ] Change the [Get started with a free quote and architecture assessment ›] and the [Contact us about your unique requirements ›] CTAs to link to the contact form instead of the copydoc
- [ ] Could you also add alt text to logos in the logo cloud using the titles on the copydoc?

**Comment by muhammad-ali-pk** [comment]
@eliman11 Thanks ELaine. All done!

**Comment by Sophie-32** [comment]
1. All images are decorative, should have alt="" 
2. I'm checking on adding one small item, should have that shortly!

**Comment by muhammad-ali-pk** [comment]
@Sophie-32 Thanks Sophie! I have added the suggestion.
Just a tiny confusion on the images. All the images except the brand logos for cloud are decorative and have `alt=""`. However, shouldn't the logos actually have the alt text, especially since brand logos usually don't count as decorative, and also, someone using Text to Speech would be needing those alt texts for the logos?

**Comment by Sophie-32** [comment]
@muhammad-ali-pk - thanks for flagging. Yes for logos our guidance is to "se the brand name (for example, ‘Microsoft Azure,’ not ‘Microsoft Azure logo’). Use empty state if the company name is used in the title or a link label."

**Comment by mattea-turic** [comment]
Hey @muhammad-ali-pk , just noticed you've got an extra asterix undet the table in "Pricing that is clear and transparent"

I'll +1 for design, though!

**Comment by immortalcodes** [comment]
image

We can remove the p-section and p-section--shallow here and just add p-section

**Comment by immortalcodes** [comment]
image

Shallow padding here

File: templates/landscape/managed.html [hunk 1]
```diff
@@ -15,250 +17,272 @@
...
-              <a href="/landscape/managed#get-in-touch">Contact us about your unique requirements</a>
-            </p>
-          </div>
+  <section class="p-section">
+    <hr class="p-rule is-fixed-width" />
+    <div class="p-section--shallow">
+      <div class="row--50-50">
+        <div class="col">
+          <h2>Bring your own identity and access management platform</h2>
+        </div>
```
Comment (immortalcodes) [inline templates/landscape/managed.html:169]:
```suggestion
    <div class="p-section--shallow">
      <div class="row--50-50">
          <hr class="p-rule" />
        <div class="col">
          <h2>Bring your own identity and access management platform</h2>
        </div>
```

File: templates/landscape/managed.html [hunk 2]
```diff
@@ -15,250 +17,272 @@
...
+      </div>
+    </div>
+    <div class="u-fixed-width u-hide--small">
+      {{ image(url="https://assets.ubuntu.com/v1/9423d6a0-bring-own-identity.png",
+            alt="",
+            width="2464",
+            height="1028",
+            hi_def=True,
+            loading="lazy",
+            attrs={"class": "p-image-container--full-width is-highlighted"}) | safe
```
Comment (immortalcodes) [inline templates/landscape/managed.html:184]: Image template needs formatting

File: templates/landscape/managed.html [hunk 3]
```diff
@@ -15,250 +17,272 @@
...
+    <div class="row--50-50">
+      <div class="col">
+        <h2>
+          Mirror Ubuntu repositories,
+          <br />
+          and distribute software internally
+        </h2>
+      </div>
+      <div class="col">
+        <div class="u-sv3">
```
Comment (immortalcodes) [inline templates/landscape/managed.html:200]: I think this is needed, it also mis-aligns the text with the heading
Please look for other similar cases as well.
image

File: templates/landscape/managed.html [hunk 4]
```diff
@@ -15,250 +17,272 @@
...
+              <th>Per Landscape virtual machine/year</th>
+              <th>Per Landscape physical machine/year</th>
+            </tr>
+          </thead>
+          <tbody>
+            <tr>
+              <th>Managed Landscape on public clouds, VMware, OpenStack or LXD</th>
+              <td>$3,099</td>
+              <td>
+                $6,861<a href="https://ubuntu.com/managed/apps">*</a>
```
Comment (immortalcodes) [inline templates/landscape/managed.html:253]:
```suggestion
                $6,861<a href="/managed/apps">*</a>
```

**Review by immortalcodes — CHANGES_REQUESTED** [review]
Thanks for the PR. Good Work!
Left minor comments

File: templates/landscape/managed.html (hunk 2 above)
Comment (muhammad-ali-pk) [reply templates/landscape/managed.html:184]: Could you please double check? I believe its formatted correctly.
Also, This is formatted with djlint, and changing it in any way reverts it back to this formatting automatically when formatting the doc.

File: templates/landscape/managed.html (hunk 3 above)
Comment (muhammad-ali-pk) [reply templates/landscape/managed.html:200]: Thanks for the careful observation Madhur!
However, the u-sv3 is needed to add a bit of spacing between the texts and the image. It doesn't affect the alignment of text with heading.

With u-sv3
image


Without u-sv3
image

File: templates/landscape/managed.html (hunk 2 above)
Comment (immortalcodes) [reply templates/landscape/managed.html:184]: image
Generally we have it like this
image

File: templates/landscape/managed.html (hunk 3 above)
Comment (immortalcodes) [reply templates/landscape/managed.html:200]: Yes, you are right, I was thinking something else!

**Comment by immortalcodes** [comment]
image

Now we have a shallow padding here so we don't need `u-sv4`

**Comment by immortalcodes** [comment]
LGTM! Please do that one change and it should be good to merge

**Review by immortalcodes — APPROVED** [review]

**Pull request merged**